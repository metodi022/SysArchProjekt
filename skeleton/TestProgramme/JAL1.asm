lui $1, 0x1234
jal end
ori $1, 0x5678
end:
ori $1, 0x1111