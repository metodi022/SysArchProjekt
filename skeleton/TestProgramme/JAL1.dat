0x3c011234
0x0c100003
0x34215678
0x34211111
