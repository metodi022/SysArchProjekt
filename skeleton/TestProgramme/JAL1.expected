@1 12341111
@1F 00400008